rosdep/python.yaml

```
@@ -7488,6 +7488,19 @@ python3-multimethod-pip:
   ubuntu:
     pip:
       packages: [multimethod]
+python3-munkres:
+  debian:
+    bullseye: [python3-munkres]
+    buster: [python3-munkres]
+  fedora: [python3-munkres]
+  opensuse: [python3-munkres]
```

Add `nixos: [python3Packages.munkre]` [python39Packages.munkres](https://search.nixos.org/packages?channel=22.05&from=0&size=50&sort=relevance&type=packages&query=munkres)

---

@kjeremy Thanks for filling this in. In the future please use the suggestion option in your review to make it trivial to merge quickly like this:

```suggestion
  nixos: [python3Packages.munkre]
  opensuse: [python3-munkres]
```